services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Utils.kt

```diff
@@ -138,3 +140,8 @@ fun String?.isValidBusinessId(): Boolean {
         false
     }
 }
+
+fun <T, C : Collection<T>> C.inspect(f: (T) -> Unit): C {
```

Comment:
Would it be better to rename this to for example applyEach or something similar? I was having a hard time to understand what inspect is supposed to do.

Reply:
Turns out there's a premade method for this: onEach. Using that instead.